index.js

```diff
@@ -68,6 +71,8 @@ const runAction = () => {
 	const release = getInput("release", true) === "true";
 	const pkgRoot = getInput("package_root", true);
 	const buildScriptName = getInput("build_script_name", true);
+  const skipBuild = getInput("skip_build") === "true"
+  const vueCliBuild = getInput("use_vue_cli") === "true";
```

Comment:
Could you please rename this variable to `useVueCli` for consistency with the option name?